apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -109,7 +118,7 @@ impl BridgeQueryPlanner {
                 data: QueryPlan { node: Some(node) },
                 usage_reporting,
             } => {
-                let subselections = node.parse_subselections(&*self.schema);
+                let subselections = node.parse_subselections(&*self.schema, operation_depth_limit);
```

Comment:
Since this is a `self` function, I don't think we need to pass this parameter. We can just access it from `self` in `plan`.